
uint8 FAILED=1
uint8 SUCCEEDED=2

int8 real_target
int8 fake_target
int8 type
---
uint8 response